db/migrate/20180205113239_move_children_postcodes_to_screener_answers.rb

```diff
@@ -0,0 +1,15 @@
+class MoveChildrenPostcodesToScreenerAnswers < ActiveRecord::Migration[5.1]
+  def up
+    C100Application.all.each do |app|
+      app.screener_answers.children_postcodes = app.children_postcodes
```

Comment:
This fails as, in my case, I have C100Application records but those are not yet linked to any `screener_answer` record so it returns `nil` and 💥 
I would personally get rid of this migration altogether as we've not released the service publicly yet and there is no harm in loosing local/test data. It might come back to bite us later.